src/components/game/ConsequenceChoice.js

```diff
@@ -102,19 +103,39 @@ class ConsequenceChoice extends Component {
     );
   }
 
+  hatBtn() {
+    const message = 'you must select a consequence before selecting pulling more items';
+    if (this.state.unselectedConsequence) {
+      return (
+        <AppContext.Consumer>
+          {context => (
+            <button
+              onClick={context.updateMessage.bind(AppProvider, message)}
```

Comment:
NOTE: this bind seems unfortunately necessary. Its the only way i've been able to figure out how to update the context with a passed in argument